iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -29,13 +29,10 @@ class ConversationMessagesCell: UICollectionViewCell {
         cv.showsVerticalScrollIndicator = false 
         return cv
     }()
-    private lazy var dataSource = ConversationMessageCellDataSource(collectionView: self.collectionView)
+    private lazy var dataSource = MessageSequenceCollectionViewDataSource(collectionView: self.collectionView)
 
     private var state: ConversationUIState = .read
 
-    /// If true, push the user messages back to prepare for a new message.
-    private var prepareForSend = false
```

Comment:
The prepare to send functionality has been moved to the message sequence datasource so it can be shared by conversations and threads.